example_test.py

```diff
@@ -18,6 +18,13 @@
 """
 Test the :py:class`pyfakefs.example` module to demonstrate the usage of the
 :py:class`pyfakefs.fake_filesystem_unittest.TestCase` base class.
+
+Fake filesystem functions like CreateFile(), CreateDirectory() or CreateLink()
```

Comment:
I now see that back ticks are inconsistently used in the docstrings.  Oh well, we'll clean that up and render it on readthedocs.com someday.